QUARKUS-2230.md

```diff
@@ -0,0 +1,21 @@
+# QUARKUS-2230 Introduce OpenShift based CI runs in upstream
+
+## Scope of the testing
+* Review the upstream workflow introduced with https://github.com/quarkusio/quarkus/pull/17674.
+  * This review will focus on the workflow itself and on test coverage of Quarkus integration surfaces with Kubernetes
+    and OpenShift APIs.
+
```

Comment:
Add something like `Propose enhancements to the workflow`, `Identify weak points`

Reply:
Updated.